js/stores/appStore.js

```diff
@@ -239,7 +239,7 @@ function windowDefaults () {
     windowOffset: 20,
     webPreferences: {
       sharedWorker: true,
-      partition: ''
+      partition: 'main-1'
```

Comment:
some confusion with electron. Normally partitions that are not prefixed with `persist:` are in-memory, but apparently '' is a special case that returns a persistent partition. It's particularly annoying because there is no way to get a matching '' in-memory partition which is needed for `spanning` extensions. At some point we will either have to change the default partition used by normal tabs or add a method that allows creating an in-memory '' partition.